osbs/build/build_request.py

```diff
@@ -502,6 +502,12 @@ def is_custom_base_image(self):
         return bool(re.match('^koji/image-build(:.*)?$',
                              self.base_image or ''))
 
+    def is_from_scratch_image(self):
+        """
+        Returns whether or not this is a build `FROM scratch`
+        """
+        return bool(re.match('^scratch(:.*)?$', self.base_image or ''))
```

Comment:
I actually think we should compare directly against the string literal 'scratch'. Since 'scratch' is a reserved name in a Dockerfile, 'scratch:latest' is not a valid 'FROM' value.

Reply:
updated